docs/setup.md

```diff
@@ -194,6 +194,14 @@ docker run -it --env-file=.env -v $PWD:/app --rm auto-gpt --gpt3only --continuou
 
 ### Run without Docker
 
+#### Create a Virtual Environment
+Create a virtual environment to run in.  
+        :::shell
+        python -m venv venvAutoGPT
+        source venvAutoGPT/bin/activate
+        pip3 install --upgrade pip
```

Comment:
The formatting of this is broken, see the preview

Reply:
@johnisanerd we'd like to get this into 0.3.2. Could you resolve this comment and update the branch?